pandas/tests/tseries/offsets/test_ticks.py

```diff
@@ -35,6 +36,39 @@ def test_delta_to_tick():
     assert (tick == offsets.Day(3))
 
 
+@given(cls=st.sampled_from(tick_classes),
```

Comment:
you should be able to remove tests which this duplicates.

Reply:
Sure; I'll convert them to explicit inputs with the `@example(...)` decorator :smile: